frontend/src/utils/types.ts

```diff
@@ -30,9 +30,11 @@ export type VideoObject = Video | VideoSerializerWithCriteria;
 export type SelectablePoll = {
   name: string;
   displayOrder: number;
+  // the path used as URL prefix, must include leading and trailing slash
   path: string;
+  // a list of id used by `SideBar` items that won't be displayed
+  disabledMenuItems?: Array<string>;
```

Comment:
These items should probably be disabled in the `PollRoutes` too.  
Maybe this list could be used for both components, and be renamed to `disabledRoutes`, or `disabledRouteIds` ?

Reply:
Sure, fixed by https://github.com/tournesol-app/tournesol/pull/660/commits/7429a5d67ab76f76028ee847f48baa2d6f035288